package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ably-common",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
```

Comment:
Because we're using GitHub this can be shortened to just:

```json
"repository": "ably/ably-common"
```

As documented [here](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository) and demonstrated by us [here](https://github.com/ably/ably-js/blob/b1ef7473d5b8ca9a3a94de543f547ccfd6e6e71d/package.json#L79).

Reply:
Nice, didn't know about this 2609d56